index.html

```diff
@@ -16,10 +19,53 @@
   </head>
 
   <body>
-    <div class="cabecalho">
-      <p style="font-size: 25px; font-weight: bold; margin: 0">
+    <div class="cabecalho" style="padding: 30px 0 0">
+      <p style="font-size: 25px; margin: 0">
         Lista de Tarefas
-      </p>
-    </div>
+  </p>
+  </div>
+
+      <div class="cabecalho" style="padding-bottom: 60px">
+          <input type="text" id="novaTarefa" placeholder="Incluir nova tarefa" />
+          <span onclick="adcionar()" class="salvar">Salvar</span>
```

Comment:
Comentamos mais abaixo, mas o nome dessa função poderia ser addNewTask ou algo similar, já que o time concordou em usar nome de função em inglês e com uma descrição mais específica, relacionada ao que faz.

Reply:
Caso não seja interessante mudar o nome da função, é preciso ajustar a chamada da mesma pois o nome está com um typo

```suggestion
          <span onclick="adicionar()" class="salvar">Salvar</span>
```